opentech/apply/funds/admin_helpers.py

```diff
@@ -69,3 +69,37 @@ def queryset(self, request, queryset):
             query = {f'{value}form__isnull': False}
             return queryset.filter(**query).distinct()
         return queryset
+
+
+class ApplicationFormButtonHelper(ButtonHelper):
+    def prepare_classnames(self, start=None, add=None, exclude=None):
+        """Parse classname sets into final css classess list."""
+        classnames = start or []
+        classnames.extend(add or [])
+        return self.finalise_classname(classnames, exclude or [])
+
+    def copy_form_button(self, pk, form_name, **kwargs):
+        classnames = self.prepare_classnames(
+            start=self.edit_button_classnames + ['icon', 'icon-plus'],
+            add=kwargs.get('classnames_add'),
+            exclude=kwargs.get('classnames_exclude')
+        )
+        return {
+            'classname': classnames,
+            'label': f'Create Copy',
```

Comment:
The button on the rounds admin says only "Copy" and they do the same thing more or less so I think they should look the same. Can you add that to this pr? Pick the look you think is best.

Reply:
Thanks for noticing. PR updated to have similar styling and text for copy button.